src/main/java/frc/robot/subsystems/swerve/PhoenixOdometryThread.java

```diff
@@ -11,19 +11,25 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 // GNU General Public License for more details.
 
+// Copyright 2024 FRC 8033 and Kevin Clark
```

Comment:
since we dont do this in other files in the codebase i dont think we should do it here. it is probably worth putting a proper license for the project, since right now we just have the one that comes auto generated with wpilib projects

Reply:
Yeah, the only reason I did it is that we have to keep the copyright notice for Mechanical Advantage since it's a derivative work, but it's so far removed from the original it's worth adding our names too. We could just do "Mechanical Advantage, Highlander Robotics, Kevin Clark" at the top line. Thoughts?